config/webpack/webpack.config.dev.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-var webpack = require('webpack');
-var config = require('./webpack.config');
+const webpack = require('webpack');
+const config = require('./webpack.config');
```

Comment:
It may be worth adding an `engines` entry in `package.json` since this won't work in Node  < 4.